src/main/kotlin/com/github/avrokotlin/avro4k/annotations.kt

```diff
@@ -28,11 +28,13 @@ annotation class AvroInline
 @Target(AnnotationTarget.PROPERTY, AnnotationTarget.CLASS)
 annotation class AvroDoc(val value: String)
 
+@Repeatable
```

Comment:
Does it make sense to have it @Repeatable here? I would rather remove it, as the user has the possibility to use the vararg.

Reply:
no repeatable anymore